lib/src/main/kotlin/com/swmansion/starknet/provider/gateway/GatewayProvider.kt

```diff
@@ -439,23 +441,38 @@ class GatewayProvider(
 
     companion object Factory {
         @JvmStatic
-        fun makeTestnetClient(): GatewayProvider {
-            return GatewayProvider(
-                "$TESTNET_URL/feeder_gateway",
-                "$TESTNET_URL/gateway",
-                StarknetChainId.TESTNET,
-            )
-        }
+        fun makeTestnetClient(testnetId: StarknetChainId = StarknetChainId.TESTNET): GatewayProvider =
```

Comment:
Default params don't translate well to java. Please overload methods instead.

Reply:
done